spec/models/vm_or_template_spec.rb

```diff
@@ -880,7 +880,7 @@
     end
 
     it "uses calculated (inline) attribute with null hardware" do
-      vm = FactoryBot.create(:vm_vmware)
+      FactoryBot.create(:vm_vmware)
       vm2 = VmOrTemplate.select(:id, :provisioned_storage).first
```

Comment:
Can you rename this `vm`?  It doesn't need to be `vm2` if there isn't a `vm` local var already